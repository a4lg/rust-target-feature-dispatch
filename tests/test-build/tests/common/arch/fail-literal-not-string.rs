pub const VALUE: i32 = target_feature_dispatch::target_feature_dispatch! {
    // ERROR: 0 is not an architecture.
    if 0 {
        (1)
    } else {
        0
    }
};
fn main() {}
