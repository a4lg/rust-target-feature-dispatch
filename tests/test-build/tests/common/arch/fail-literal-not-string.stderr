error[E0565]: literal in `cfg` predicate value must be a string
 --> tests/common/arch/fail-literal-not-string.rs:3:8
  |
3 |     if 0 {
  |        ^
